qemu/tests/qemu_guest_agent.py

```diff
@@ -820,6 +851,24 @@ class QemuGuestAgentBasicCheckWin(QemuGuestAgentBasicCheck):
     """
     Qemu guest agent test class for windows guest.
     """
+
+    @error_context.context_aware
+    def gagent_uninstall(self, session, vm, *args):
```

Comment:
This can be moved into `QemuGuestAgentBasicCheck`.

Reply:
@luckyh 
I update all those comments. Pls have a check. 
Thanks
 (1/1) smp_2.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Win10.i386.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.gagent_install_uninstall: PASS (798.07 s)